### PR TITLE
Rebuild to pick up latest fontconfig changes

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
A rebuild seems to be necessary to pick up https://github.com/conda-forge/fontconfig-feedstock/pull/53, see https://github.com/conda-forge/fontconfig-feedstock/pull/53#issuecomment-1062486868

This is needed for:
https://github.com/conda-forge/poppler-feedstock/pull/124
and
https://github.com/conda-forge/gdal-feedstock/pull/611

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Windows error:
```
LINK : fatal error LNK1181: cannot open input file '%PREFIX%\Library\lib\xml2.lib'
make[1]: *** [release/cairo.dll] Error 157
Makefile.win32:16: recipe for target 'release/cairo.dll' failed
make[1]: Leaving directory '/d/bld/cairo_1646836154097/work/src'
make: *** [cairo] Error 2
Makefile.win32:12: recipe for target 'cairo' failed
```

**Comment by pkgw** [comment]
The Windows error is related to the fact that the Meson switch changes `fontconfig` from libxml2 to expat. I'm locally testing a patch to fix this — note that `bld.bat` explicitly includes `xml2.lib` in the link line. We might just be able to take it out? I don't see any XML deps in Cairo proper.

**Comment by xylar** [comment]
@pkgw, let me know how the local test goes and feel free to just make the fix on this branch if you like.

**Comment by pkgw** [comment]
@xylar Seems to work! Update pushed.

**Comment by xylar** [comment]
The good news is also that the OSX build seems to have worked so hopefully this will be end of the `fontconfig` dependency chain for `gdal`.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -91,8 +91,11 @@ about:
   home: http://cairographics.org/
   dev_url: https://gitlab.freedesktop.org/cairo/cairo
   doc_url: https://www.cairographics.org/documentation/
-  license: LGPL 2.1 or MPL 1.1
-  license_file: COPYING
+  license: LGPL-2.1-only or MPL-1.1
+  license_file: 
+    - COPYING
+    - COPYING-LGPL-2.1
+    - COPYING-MPL-1.1
```
Comment (xylar) [inline recipe/meta.yaml:98]: @conda-forge/cairo, please make sure you're okay with these changes.  I think they are correct but it never hurts to have another pair of eyes on a thing like this.

**Comment by xylar** [comment]
Yay, the tests finally passed!  @conda-forge/cairo, this is ready to review and hopefully merge.

File: recipe/meta.yaml (hunk 1 above)
Comment (pkgw) [reply recipe/meta.yaml:98]: Yes, this change looks correct to me. Thanks!

**Comment by xylar** [comment]
thanks very much, @pkgw!

**Pull request merged**